adhocracy-plus/templates/account/signup.html

```diff
@@ -6,6 +6,10 @@
 {% block content %}
     <h1>{% translate "Register" %}</h1>
 
+    {% if settings.a4_candy_cms_settings.ImportantPages.registration.url %}
+        <a href="{{ settings.a4_candy_cms_settings.ImportantPages.registration.url }}" target="_blank">{% blocktranslate %}Why register?{% endblocktranslate %}</a>
+    {% endif %}
+
```

Comment:
It needs an extra check for the page to be published and not in draft mode. As it is now the link appears even if  the register page is a draft

Reply:
hmm, I guess that raises the question if we need this check at all, we don't have a check for the other pages (e.g. in the checkbox for the data protection page, or the footer), I just thought it's nice to hide it. The story doesn't say it should only be shown if the page exists. I will look into how easy it is to check if the page is draft

Reply:
@m4ra added a check

Reply:
thanks! can we have a nice wrap of this long variable of the important page using the [with](https://docs.djangoproject.com/en/5.0/ref/templates/builtins/#with) block. and a commit squash?

Reply:
done